pkg/services/authn/clients/grafana_test.go

```diff
@@ -40,13 +40,13 @@ func TestGrafana_AuthenticateProxy(t *testing.T) {
 				proxyFieldEmail:  "email@email.com",
 			},
 			expectedIdentity: &authn.Identity{
-				OrgRoles:   map[int64]org.RoleType{1: org.RoleViewer},
-				Login:      "test",
-				Name:       "name",
-				Email:      "email@email.com",
-				AuthModule: "authproxy",
-				AuthID:     "test",
-				Groups:     []string{"grp1", "grp2"},
+				OrgRoles:        map[int64]org.RoleType{1: org.RoleViewer},
+				Login:           "test",
+				Name:            "name",
+				Email:           "email@email.com",
+				AuthenticatedBy: "authproxy",
```

Comment:
super nit: here and below could use `login.AuthProxyAuthModule` instead of `"authproxy"`.

Reply:
Done